### PR TITLE
fix kustomize scan breakage on windows

### PR DESCRIPTION
**Pull request by dev-gaur**

### CODE REVIEW

File: pkg/iac-providers/kustomize/v3/load-dir_test.go [hunk 1]
```diff
@@ -4,13 +4,16 @@ import (
...
 	"reflect"
+	"strings"
 	"syscall"
 	"testing"
 
 	"github.com/accurics/terrascan/pkg/iac-providers/output"
 	"github.com/accurics/terrascan/pkg/utils"
 )
 
+const kustomizeErrPrefix = "error from kustomization."
```
Comment (patilpankaj212) [inline pkg/iac-providers/kustomize/v3/load-dir_test.go:15]: can this be an `error` variable?

File: pkg/iac-providers/kustomize/v3/load-dir_test.go [hunk 2]
```diff
@@ -154,20 +157,20 @@ func TestLoadKustomize(t *testing.T) {
...
-			wantErr:  errorFromKustomize,
+			wantErr:  fmt.Errorf(kustomizeErrPrefix),
 		},
 	}
 
 	for _, tt := range table {
 		t.Run(tt.name, func(t *testing.T) {
 			_, gotErr := LoadKustomize(tt.basepath, tt.filename)
-			if !reflect.DeepEqual(gotErr, tt.wantErr) {
+			if (tt.wantErr != nil && !strings.HasPrefix(gotErr.Error(), tt.wantErr.Error())) && !reflect.DeepEqual(gotErr, tt.wantErr) {
```
Comment (patilpankaj212) [inline pkg/iac-providers/kustomize/v3/load-dir_test.go:173]: we should introduce a new bool in the test struct for the prefix check!

**Review by patilpankaj212 — CHANGES_REQUESTED** [review]

File: pkg/iac-providers/kustomize/v3/load-dir_test.go (hunk 1 above)
Comment (dev-gaur) [reply pkg/iac-providers/kustomize/v3/load-dir_test.go:15]: it isnt exacty an error message.. but a prefix

**Review by patilpankaj212 — APPROVED** [review]
Verify failing tests on windows

**Review by kanchwala-yusuf — APPROVED** [review]

**Comment by kanchwala-yusuf** [comment]
@dev-gaur can you just run `go mod tidy` on this PR branch?

**Pull request merged**